test/ed25519.spec.js

```diff
@@ -35,6 +35,27 @@ describe('ed25519', () => {
     })
   })
 
+  it('generates a valid key from seed', (done) => {
```

Comment:
Could you please add a test that generating twice from the same seed actually results in the same key?